### DISCUSSION TITLE
Logging out on 'other devices/browsers' or log out everywhere

### DISCUSSION THREAD

**Discussion by atomtigerzoo** [supabase/supabase#30162] [Questions] [answered]
Hi! I really like the supabase idea and service, but I am having trouble wrapping my head around the auth system and the never ending logins.

Given scenario:
I am using a computer in a library or lets say at a friends house. I log in with supabase, do some work and leave the place. Or even worse: the computer shuts down because of something (electricity loss, times up, just forgot it, ...) and I have to leave without logging out. Bot the only thing I can do - or have to do - is wait until I can get back to the computer (next day, after the weekend, ...) and log out, hoping no other person used my logged-in account to do stuff. I can call IT-support or beg my friend to do it, hoping the best. (Good friends just do it without looking, I know...)

Am I right, that there is now way to end all sessions? Like others have implemented the way to delete other login sessions on different devices/browsers or just log out/end all sessions. I have found SQL queries to run or server functions with warnings, but this cannot be the answer to such a crucial thing.

I know that keeping users in for a long time can be a good thing and can be wished for. But signing out all devices/browsers/users-sessions also sounds good and more secure, than keeping session forever-ever and "delete the session/cache on the other devices".

Is this a feature that is planned somewhere or is it just a design descision made for supabase-auth?

**Comment by silentworks** [comment, marked as answer]
With a Pro account you can setup time-boxed sign in's. This way the user has to sign in again after a certain amount of time. Also you can set your JWTs to be more short lived and turn off refreshing in the client library. It's up to the developer of the app to decide how they want it to work, Supabase provides the options for you to code it how you see best. 

![image](https://github.com/user-attachments/assets/bed49f8d-3f6b-4d34-9df5-69a6aa1a755f)

**Comment by atomtigerzoo** [comment]
Thank you for answer. Too sad this option is only for payed accounts.
